src/NetworkTestWidget/RegionResult/getTooltipContent.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { rows } from '../../ResultWidget/rows';
+import { TestResults } from '../../types';
+import { Typography as TooltipTypography } from '../../ResultWidget/rows/shared';
+
+export default function getTooltipContent(result: TestResults): React.ReactElement[] {
+  const warnings = rows
+    .map((row) => {
+      const warning = row.getWarning?.(result);
+      return warning ? row.tooltipContent?.[warning] : null;
+    })
+    .filter((content) => content !== null)
+    .map((content, i) => <React.Fragment key={i}>{content}</React.Fragment>); // Adding keys to suppress dev warning
```

Comment:
I think keys should be unique. Should we prefix this?

Reply:
The index should be unique enough.  React keys just need to be unique to the array they are in. They don't need to be globally unique or anything. 

Also, in this case the keys aren't actually needed. This array is never going to be updated and re-rendered. The only reason the keys are added here is to suppress React's warning about not using keys. It's not an ideal situation, but I can't think of a better way.

Reply:
Got it.